src/rt/sched/schedulerthread.h

```diff
@@ -356,16 +356,16 @@ namespace verona::rt
             break;
         }
 
-        Systematic::cout() << "Schedule cown: " << cown << " ("
-                           << cown->get_epoch_mark() << ")" << std::endl;
-
         // Administrative work before handling messages.
         if (!prerun(cown))
         {
           cown = nullptr;
           continue;
         }
 
+        Systematic::cout() << "Schedule cown: " << cown << " ("
```

Comment:
Nit: This change is not described in the commit message.

Reply:
This message would previously give garbage information in the case that the `cown` is actually the token cown used for fairness. This message should be after the prerun step that checks for the token.